TestAssets/LockFileSamples/ExportFileSamples/valid/project.lock.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
I'm not sure 'Samples' is the right word in the directory names. How about just LockFiles? Also, ExportFile~~Sample~~s seem like peers to LockFiles, not a specialization of them.